### DISCUSSION TITLE
Translation to Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by adambravo79** [bcholmes/StarTrek2d20#207] [Ideas]
Hi man, first of all, many thanks for the incredible work.
Second, I am just here to suggest an Brazilian Portuguese translation.
The book got published about a year ago and I believe many brazlians can benefit from all these amazing tools if we had the most part translated to Portuguese.
I'll add a column in .csv file as soon I send this message.

Thanks again!

**Comment by bcholmes** [comment]
Wow. You got through that really quickly. I've updated the site with your translation text. Check it out.

**Reply by adambravo79** [reply]
Everything is ok, just a few strings that apparently weren't in the spreadsheet that still have English names, they must be hard coded.
But most of the screens are already usable and now that I can see where each string is going I'll be able to improve some texts (Portuguese is a very annoying  language, full of plurals and genders)... but everything will be fine.
Thanks!
